spec/models/step_by_step_page_spec.rb

```diff
@@ -314,6 +314,12 @@
       step_by_step_page.mark_draft_updated
 
       expect(step_by_step_page.can_be_published?).to be false
+    end
+
+    it 'cannot be published if all steps do not have content' do
```

Comment:
Is it worth writing an additional test which has two steps - one with content, one without - and ensures that it cannot be published?

Reply:
This is covered by the first step, but I'll update the test name to make that clear.